.github/workflows/auto-version.yml

```
@@ -0,0 +1,61 @@
+          PATCH=$((PATCH + 1))
+          NEW_VERSION="$MAJOR.$MINOR.$PATCH"
+          
+          # Update package.json
+          npm version $NEW_VERSION --no-git-tag-version
+          echo "new_version=$NEW_VERSION" >> $GITHUB_OUTPUT
+
+      - name: Commit version bump
+        run: |
+          git add package.json package-lock.json
+          git commit -m "chore: bump version to v${{ steps.increment_version.outputs.new_version }} [skip ci]"
+          git push origin dev 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)